dsf-bpe/dsf-bpe-process-base/src/main/java/org/highmed/dsf/fhir/variables/SerializerPlugin.java

```diff
@@ -11,29 +15,28 @@
 public class SerializerPlugin implements ProcessEnginePlugin
 {
 	@SuppressWarnings("rawtypes")
-	private final List<TypedValueSerializer> serializer;
+	private List<TypedValueSerializer> serializer;
```

Comment:
This is not relevant any more, because the Serializer Plugin was removed. But it was probably a mistake by my side, because I played around with multiple Serializer Plugins.